x-pack/plugins/uptime/server/plugin.ts

```diff
@@ -77,7 +77,8 @@ export class Plugin implements PluginType {
       this.syntheticService = new SyntheticsService(
         this.logger,
         this.server,
-        this.server.config.service
+        this.server.config.service,
+        this.initContext.env.packageInfo.version
```

Comment:
might be worth moving this to the `UptimeServerSetup` object.  And then access it from that object, this.server in the synthetics class.